src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -343,12 +345,20 @@ const formConfig = {
           schema: socialBehaviorChanges.schema,
         },
         mentalHealthChanges: {
-          title: 'Additional Remarks - Physical Health Changes',
+          title: 'Additional Remarks - Mental Health Changes',
           path: 'new-disabilities/ptsd-781a-mental-changes',
           depends: isAnswering781aQuestions(0),
           uiSchema: mentalHealthChanges.uiSchema,
           schema: mentalHealthChanges.schema,
         },
+        workBehaviorChanges: {
+          title: 'Additional Remarks - Behavior Changes at Work',
+          path: 'new-disabilities/ptsd-781a-work-changes',
+          depends: formData =>
```

Comment:
Can we set this `depends` up just like the `mentalHealthChanges` and get rid of `isAnsweringPtsdForm` (in utils as well).

Reply:
changed!